app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+    enum role: { client: 0, management: 1 }
```

Comment:
I would recommend creating a UserRole model and associate it with User
User belongs_to :user_role
UserRole has_many :users

it will provide you with more flexibility in the future, if user roles need any logic or parameters associated with them

Reply:
If you go with that you will need to create required roles in `db/seeds.rb`

Reply:
For know just for simplicity I think I'll stick with it as it is